[AHM] OpenGov for Kusama AH

karolk91 opened the pull request:
Merging https://github.com/polkadot-fellows/runtimes/pull/776 to https://github.com/polkadot-fellows/runtimes/pull/862

bkontur committed:
Initial copy of Kusama OpenGov to AHK (commit: 7bccafeee512e90126843e988a6875be635eaa56 (tag 1.6.0))

bkontur committed:
Add some weights files and deps

bkontur committed:
Add pallet-preimage

bkontur committed:
Add scheduler + TODOs

bkontur committed:
Add pallet-parameters + TODOs

bkontur committed:
Add Treasury/Boundties setup + TODOs

bkontur committed:
Governance integration tests on Kusama (for OpenGov on AH)

bkontur committed:
OpenGov AH accepts RC Fellows voice for `WhitelistOrigin`

bkontur committed:
Kusama RC accepts AssetHubLocation as a root

bkontur committed:
All Kusama system paras accept AssetHubLocation as root

bkontur committed:
// TODO: split GovernanceOnRcLocation and GovernanceOnAhLocation - because we don't send migration start to other system paras

bkontur committed:
more TODOs

bkontur committed:
Fix/split `GovernanceLocation` for system paras (Kusama).

bkontur committed:
GeneralAdmin/StakingAdmin/Fellows origin for OpenGov on AH (Kusama)

karolk91 committed:
Merge remote-tracking branch 'upstream/main' into kk-bko-ahm-kusama-governance-tweaks3

karolk91 committed:
Missing common.rs file

karolk91 committed:
Merge remote-tracking branch 'upstream/main' into kk-bko-ahm-kusama-governance-tweaks3

karolk91 committed:
Align TreasuryPaymaster with PAH from dev-asset-hub-migration branch

karolk91 committed:
Formatting fixes

karolk91 committed:
Fix Clippy and taplo errors

karolk91 committed:
Formatting fixes

karolk91 committed:
missing asset-hub-kusama/src/weights/pallet_asset_rate.rs added

bkontur committed:
Merge pull request #20 from karolk91/kk-bko-ahm-kusama-governance-tweaks2

[WIP] sync AHM branch with main

karolk91 committed:
Fix encointer tests

karolk91 committed:
Fix features propagation reported by zepter

karolk91 committed:
set global GovernanceLocation to AssetHub

bkontur committed:
More clean up

karolk91 committed:
Cleanup

bkontur committed:
Nit

bkontur committed:
More nits

karolk91 committed:
Missing improvement for gov tests

karolk91 committed:
Merge remote-tracking branch 'upstream/main' into bko-ahm-kusama-governance-tweaks

karolk91 committed:
Merge remote-tracking branch 'muharem/muharem-ahm-kusama' into kk-muharem-ahm-kusama-bko-ahm-kusama-governance-tweaks

bkontur commented on system-parachains/asset-hubs/asset-hub-kusama/src/governance/origins.rs:
```diff
@@ -60,18 +58,44 @@ pub mod pallet_custom_origins {
+		/// Origin commanded by rank 8 of the Polkadot Fellowship and with a success of 8.
+		Fellowship8Dan,
+		/// Origin commanded by rank 9 of the Polkadot Fellowship and with a success of 9.
+		Fellowship9Dan,
```
I think Muharem intentionally removed those, because they stay on the Kusama relaychain
```suggestion
```

bkontur commented on system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:
```diff
@@ -1568,8 +1631,11 @@ mod benches {
 		[pallet_preimage, Preimage]
 		[pallet_proxy, Proxy]
+		[pallet_scheduler, Scheduler]
+		[pallet_referenda, Referenda]
```
duplicate
```suggestion
```

bkontur commented on system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:
```diff
@@ -1581,6 +1647,7 @@ mod benches {
 		[pallet_timestamp, Timestamp]
 		[pallet_treasury, Treasury]
 		[pallet_transaction_payment, TransactionPayment]
+		[pallet_whitelist, Whitelist]
```
duplicate
```suggestion
```

karolk91 committed:
Update system-parachains/asset-hubs/asset-hub-kusama/src/governance/origins.rs

Co-authored-by: Branislav Kontur <bkontur@gmail.com>

karolk91 committed:
Update system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

Co-authored-by: Branislav Kontur <bkontur@gmail.com>

karolk91 committed:
Update system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

Co-authored-by: Branislav Kontur <bkontur@gmail.com>

karolk91 committed:
Remove fellows related stuff from AH

karolk91 committed:
Formatting fixes

karolk91 committed:
Remove unused module, replaced by polkadot-sdk's impl

muharem committed:
[AHM] Integrate migration for Kusama (#862)

Integrate migration for Kusama

The PR introduces migrating pallets to the Kusama Asset Hub runtime.
Additionally adjusting and integrating the migrator pallets for the
Kusama Relay and Asset Hub chains.

---------

Signed-off-by: Oliver Tale-Yazdi <oliver.tale-yazdi@parity.io>
Co-authored-by: Oliver Tale-Yazdi <oliver.tale-yazdi@parity.io>

karolk91 committed:
Merge remote-tracking branch 'muharem/muharem-ahm-kusama' into kk-muharem-ahm-kusama-bko-ahm-kusama-governance-tweaks

karolk91 committed:
Merge remote-tracking branch 'upstream/oty-dev-asset-hub-migration-2506' into kk-muharem-ahm-kusama-bko-ahm-kusama-governance-tweaks

karolk91 committed:
Update system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

Co-authored-by: Branislav Kontur <bkontur@gmail.com>

karolk91 committed:
Fmt + unused dep
